### PR TITLE
Release v0.23.0

### PR DESCRIPTION
**Pull request by sjcobb**
Release notes for v0.23.0, steps taken:
- update VERSION to 0.23.0
- run cmd: `make generate-changelog`
- manual cleanup of CHANGELOG.md
- run cmd: `make bump-version`
- commit changes

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).
- [x] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+- [BUGFIX] time series migration add unit inside y_axis (#952)
+- [BUGFIX] round small decimal values when abbreviate is set (#946)
+- [BUGFIX] Fix empty dashboard spec that wasn't rejected (#931)
+- [BUGFIX] Fix discard changes confirmation dialog showing up even when there is no change (#929)
+- [BUGFIX] Fix increased metadata.version (#925)
+- [BUGFIX] Fix API error EOF that happens when decoding multiple times the body from a middleware (#923)
+- [BREAKINGCHANGE] Move panel padding to plugins (#942)
+- [BREAKINGCHANGE] Add a consistent JSON tab for every panel plugin (#922)
+- [BREAKINGCHANGE] remove deprecated top-level `unit` in time series panel (#920)
+
```
Comment (Nexucis) [inline CHANGELOG.md:23]: I think we add we removed the support of etcd as the main database

Comment (sjcobb) [reply CHANGELOG.md:23]: Hmm is that merged yet, I've seen this [draft PR](https://github.com/perses/perses/pull/961) but is there another previous one? Also, what exactly will that mean, can we keep the same workflow for local dev using file system / local_db?

Comment (sjcobb) [reply CHANGELOG.md:23]: Oh nvm I see what you mean now in https://github.com/perses/perses/pull/933
I'll add that BREAKINGCHANGE note

Comment (Nexucis) [reply CHANGELOG.md:23]: ah yes sorry, I forgot the PR linked. Cool to see you found it !

Comment (sjcobb) [reply CHANGELOG.md:24]: Breaking change note is added now!

**Review by Nexucis — APPROVED** [review]

**Review by eunicorn — APPROVED** [review]

**Review by cndonovan — APPROVED** [review]
lgtm! bunch of cool stuff coming

**Review by juliepagano — APPROVED** [review]
LGTM! Thanks for cutting a release. 🏆

**Pull request merged**